apps/recommendation/api/src/huggy/core/model_selection/model_configuration/configuration.py

```diff
@@ -133,36 +134,17 @@ def get_user_status(self, user: UserContext, model_origin: str) -> ForkOut:
         )
 
     def get_offer_status(
-        self, offer: Offer, offers: list[Offer], model_origin: str
+        self, input_offers: Optional[list[Offer]], model_origin: str
     ) -> ForkOut:
         """
         Get model status based on Offer interactions
 
         """
-
-        if offers:
-            return ForkOut(
-                copy.deepcopy(self.warm_start_model),
-                reco_origin="algo",
-                model_origin=model_origin,
-            )
-
-        if not offer.found:
-            return ForkOut(
-                copy.deepcopy(self.cold_start_model),
-                reco_origin="unknown",
-                model_origin=model_origin,
-            )
-        if self.bookings_count is not None:
-            if offer.booking_number >= self.bookings_count:
-                return ForkOut(
-                    copy.deepcopy(self.warm_start_model),
-                    reco_origin="algo",
-                    model_origin=model_origin,
-                )
+        # No cold start logic for similar offer.
```

Comment:
This part will be removed afterwards with the recommendation logic as it is legacy.
Current configurations for similar offers logic don't take into account the cold_start logic